src/katsdpcontroller/product_controller.py

```diff
@@ -807,7 +806,7 @@ async def _multi_request(
                 if isinstance(result, Exception):
                     raise result
         if non_critical:
-            raise FailReply(f"No katcp connection to some nodes: {non_critical}")
+            logger.debug("multi_request: no katcp connection to some nodes: %s", non_critical)
```

Comment:
Don't you want to make this at least warning level?

Reply:
You'll get a warning when the node dies. You don't need a reminder warning every 5 seconds (every time delays get updated).